pytorch/src/main/java/org/bytedeco/pytorch/presets/torch.java

```diff
@@ -1852,11 +1959,13 @@ We need either to put an annotation info on each member, or javaName("@NoOffset
                ))
                .put(new Info("c10::util::get_type_index<std::string>").javaNames("get_type_index_string"))
                .put(new Info("at::TensorBase::data_ptr<int8_t>").javaNames("data_ptr_char"))
+               .put(new Info("at::TensorBase::data_ptr<uint8_t>").javaNames("data_ptr_byte"))
                .put(new Info("at::TensorBase::data_ptr<int16_t>").javaNames("data_ptr_short"))
                .put(new Info("at::TensorBase::data_ptr<int>").javaNames("data_ptr_int"))
                .put(new Info("at::TensorBase::data_ptr<int64_t>").javaNames("data_ptr_long"))
                .put(new Info("at::TensorBase::data_ptr<float>").javaNames("data_ptr_float"))
                .put(new Info("at::TensorBase::data_ptr<double>").javaNames("data_ptr_double"))
+               .put(new Info("at::Tensor::item<bool>").javaNames("item_bool"))
                .put(new Info("at::Tensor::item<int8_t>").javaNames("item_char"))
                .put(new Info("at::Tensor::item<int16_t>").javaNames("item_short"))
                .put(new Info("at::Tensor::item<int>").javaNames("item_int"))
```

Comment:
Let's add data_ptr_bool() and item_byte() for consistency

Reply:
You're right `about data_ptr_bool`.
But `item<uint8_t>`, like said [in this thread](https://github.com/bytedeco/javacpp-presets/issues/1321#issuecomment-1768937456), means "get the unique element of this tensor and return it as a `uint8_t`".  Since we don't have `uint8_t` in Java, it doesn't make much sense. If we want an `item_byte` that returns a Java `int`, we would add a `javaNames("item_byte")` to `item<int>`, this would be weird.

Reply:
Just add it, it doesn't matter if it's "weird", it does what it's supposed to do!

Reply:
BTW,  I don't think item() will allow you to cast a uint8_t to int, so just return a byte that's fine.

Reply:
It does. `item<int>` is just a macro to `item().toInt()` where `item()` returns a `Scalar`.

Reply:
Right, so for an uint_8 Scalar, toInt() fails, in my experience, so don't do that. Just return a byte, that's fine

Reply:
Just tested, and it works. So we can do either, as you wish. In both case for me `item_byte` is meaningless and is just a placeholder to match the `data_ptr` methods.